### DISCUSSION TITLE
doris过一段时间后查询插入超时，重启后恢复，一段时间后又重复出现问题

### DISCUSSION THREAD

**Discussion by weiyuqing** [apache/doris#29050] [A - General / Q&A]
出问题的时候，各节点状态都正常，测试表数据不超过10条。
报错信息如下：
ERROR 1105 (HY000): errCode = 2, detailMessage = query timeout

ERROR 1105 (HY000): RpcException, msg: io.grpc.StatusRuntimeException: UNAVAILABLE: Network closed for unknown reason

**Comment by LemonLiTree** [comment]
这个是可能是be 挂 了 ，哪个版本？

**Comment by weiyuqing** [comment]
be节点状态一直是正常的，1.2.7和2.0.3都测试过，同样的问题。
环境：移动云  ubuntu22、ubuntu20、centos7.9都是一样的问题。
日志见附件，还请帮忙分析。


***@***.***
 
发件人： Liqf
发送时间： 2023-12-27 11:42
收件人： apache/doris
抄送： rabbit; Author
主题： Re: [apache/doris] doris过一段时间后查询插入超时，重启后恢复，一段时间后又重复出现问题 (Discussion #29050)
这个是可能是be 挂 了 ，哪个版本？
―

**Reply by llaimm** [reply]
我们也是移动云，同样出现这种问题，查询几次或者重启BE就好了。很奇怪，社区跟我说可能是虚拟机超售了。

**Comment by LemonLiTree** [comment]
建表数据 查询 方便发下吗

**Comment by baofengzhouyu** [comment]
这个解决了吗？ 我也遇到相同的问题。 版本：2.1.0

**Reply by Power098** [reply]
我也遇见了相同的问题，同2.1.0版本

**Reply by Muzi999** [reply]
me too

**Reply by Arthur97662** [reply]
2.1.0都不行？那我还升级么？我状况也类似==

**Comment by moxiaokai** [comment]
这个问题有修复的版本吗？最近也遇到了同样问题：2.0.12版本

**Reply by maruyue** [reply]
中文问题，建议到ask.selectdb.com中文doris技术论坛上提问。
